src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -1937,7 +1937,26 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 }
                 break;
             }
-
+            case NI_Sve_Compute16BitAddresses:
```

Comment:
I'm tempted to suggested all these cases should be combined and the constant should be: `( id -NI_Sve_Compute8BitAddresses )`. With an assert to confirm `NI_Sve_Compute8BitAddresses` to `NI_Sve_Compute64BitAddresses` is sequential.

But also happy as it is

Reply:
yes, was not happy with 4 different case statements. Thanks for the suggestion.